src/client/mcp-client.ts

```
@@ -31,12 +31,12 @@ export class MCPClient implements IMCPClient {
     private serverSpawned = false;
     private serverPid: number | null = null;
     private serverAlias: string | null = null;
-    private timeout: number | undefined = undefined;
+    private timeout: number; // Changed to number, as Zod default ensures it
 
     constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)